src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -80,16 +76,14 @@ protected final void onStartup() {
 
     @SuppressWarnings("unchecked")
     private void initializeDrivingVariants() {
-        // Need to discover the right codec for the driving source of variants manually, since we are
-        // treating it specially (separate from the other sources of Features in our FeatureManager).
-        final FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingVariantFile, VariantContext.class);
-
-        //This is the data source for the driving source of variants, which uses a cache lookahead of FEATURE_CACHE_LOOKAHEAD
-        drivingVariants = new FeatureDataSource<>(drivingVariantFile, (FeatureCodec<VariantContext, ?>)codec, "drivingVariants", FEATURE_CACHE_LOOKAHEAD);
 
         //Add the driving datasource to the feature manager too so that it can be queried. Setting lookahead to 0 to avoid caching.
         //Note: we are disabling lookahead here because of windowed queries that need to "look behind" as well.
```

Comment:
Can you move this comment above the line it's actually talking about? (`features.addToFeatureSources(0, drivingVariantsFeatureInput, VariantContext.class);`)